test/cpp/interop/interop_client.cc

```diff
@@ -1052,6 +1052,32 @@ bool InteropClient::DoChannelSoakTest(int32_t soak_iterations) {
   return true;
 }
 
+bool InteropClient::DoLongConnectionTest(int32_t soak_iterations, 
+                                         int32_t iteration_interval) {
+  gpr_log(GPR_DEBUG, "Sending %d RPCs...", soak_iterations);
+  GPR_ASSERT(soak_iterations > 0);
+  GPR_ASSERT(iteration_interval > 0);
+  SimpleRequest request;
+  SimpleResponse response;
+  int num_failures = 0;
+  for (int i = 0; i < soak_iterations; ++i) {
+    gpr_log(GPR_DEBUG, "Sending RPC number %d...", i);
+    if (!PerformLargeUnary(&request, &response)) {
+      gpr_log(GPR_ERROR, "Iteration %d failed.", i);
+      num_failures++;
+    }
+    gpr_sleep_until(gpr_time_add(
+        gpr_now(GPR_CLOCK_REALTIME), gpr_time_from_seconds(
+            iteration_interval, GPR_TIMESPAN)));
+  }
+  gpr_log(GPR_DEBUG, "long_connection test done.");
```

Comment:
nit: can we also log the total number of failures here?

Reply:
Done.